### ISSUE TITLE
Feature request: turn order alteration

### ISSUE THREAD

**Issue by aeschenkarnos** [Cockatrice/Cockatrice#3795]
In light of the existence of [[Aeon Engine]] it seems that future cards may alter turn order. I suggest that a game owner should be able to arbitrarily reorder the turn order by dragging player names around, including a one-click "reverse all turn orders" which could be put under the kick/PM/friend context menu.

Some players of EDH seem to expect that their turns should go in the dice order rather than "winner first then standard order" which is current best practice. Letting the game owner re-order the turn order would implement this, for those who want to.

**Comment by skwerlman** [comment]
duplicate #3062

**Comment by aeschenkarnos** [comment]
I disagree, #3062 asks for *only* randomization of the order. Though in a 4-player game you could just repeatedly randomize the order and get the result you want after a dozen tries or so, it's more tedious than a simple rearrange. I do agree that randomization is also useful (though can be done with die rolls and then intentional arrangement).

**Comment by ctrlaltca** [comment]
[[Aeon Engine]] fixed by #3802.
Being able to arbitrarily reorder the turn is not possible with the current way turn order is handled, but maybe will be implemented in the future if needed.

**Comment by ZeldaZach** [comment]
I still want this feature, but it's a low priority. It would require a lot of changes to the game definitions system and player code, which is a lot of work. I think it'll happen one day, but just not today.
